file/file_small_test.go

```diff
@@ -71,29 +60,15 @@ func TestFilePublisher(t *testing.T) {
 			})
 
 			Convey("So config policy should be a cpolicy.ConfigPolicy type", func() {
-				So(configPolicy, ShouldHaveSameTypeAs, &cpolicy.ConfigPolicy{})
+				So(configPolicy, ShouldHaveSameTypeAs, plugin.ConfigPolicy{})
 			})
 		})
-		Convey("Publish content to file", func() {
-			var buf bytes.Buffer
-			enc := gob.NewEncoder(&buf)
-			enc.Encode(mockMts)
 
-			config := make(map[string]ctypes.ConfigValue)
-			config["file"] = ctypes.ConfigValueStr{Value: "/tmp/pub.out"}
-
-			Convey("invalid contentType", func() {
-				err := fp.Publish("", buf.Bytes(), config)
-				So(err, ShouldResemble, errors.New("Unknown content type ''"))
-			})
-			Convey("empty content", func() {
-				err = fp.Publish(plugin.SnapGOBContentType, []byte{}, config)
-				So(err, ShouldNotBeNil)
-			})
-			Convey("successful publishing", func() {
-				err = fp.Publish(plugin.SnapGOBContentType, buf.Bytes(), config)
-				So(err, ShouldBeNil)
-			})
+		Convey("Publish content to file", func() {
```

Comment:
These cases were directly correlated with legacy RPC: invalid and empty content. Reproducing this behaviour with lib-go is impossible, thus the cases were dropped.